CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Fixed
 
 - Use `context.Background()` as default context instead of nil in `go.opentelemetry.io/contrib/bridges/otellogr`. (#6527)
+- Convert Prometheus histogram to non-cumulative otel histogram in `go.opentelemetry.io/contrib/bridges/prometheus`. (#6685)
```

Comment:
```suggestion
- Convert Prometheus histogram to a delta histogram in `go.opentelemetry.io/contrib/bridges/prometheus`. (#6685)
```

Reply:
I prefer the "non-cumulative" wording.  I would suggest
```suggestion
- Convert Prometheus histogram buckets to non-cumulative otel histogram buckets in `go.opentelemetry.io/contrib/bridges/prometheus`. (#6685)
```

Reply:
Thanks. Done